modular_skyrat/modules/modular_items/lewd_items/code/lewd_items/venom_milker.dm

```diff
@@ -0,0 +1,115 @@
+/obj/item/reagent_containers/venom_milker
+	name = "\improper venom siphon"
+	desc = "A commercial grade venom siphon, made for use on larger - typically human sized - animals. \
+	Has a built-in reagent neutralizer that inhibits the effects of most extracted toxins for safe handling, but it cannot be guaranteed to work."
+
+	icon_state = "venom_milker"
+	icon = 'modular_skyrat/modules/modular_items/lewd_items/icons/obj/lewd_items/lewd_items.dmi'
+
+	volume = 20
+
+	reagent_flags = OPENCONTAINER
+	w_class = WEIGHT_CLASS_SMALL
+
+/obj/item/reagent_containers/venom_milker/Initialize(mapload)
+	. = ..()
+
+	var/milkable_string = /datum/preference/choiced/venomous_bite_venom::milkable_string
+	if (length(milkable_string))
+		desc += span_notice("\nThe following reagents cannot be filtered by the neutralizer: [milkable_string]")
```

Comment:
Needs better naming for milkable_string as this is for venom that can't exactly be extracted into valid venom

Reply:
better?